week-1/1-exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,4 +1,12 @@
 // Declare your function here
+function createLongGreeting(name, age){
+    let nameMess = `Hello, my name is ${name} `;
+    let ageMess = `and i'm ${age} years old`;
+    return nameMess + ageMess;
```

Comment:
This works well but I have a small tip on style for you.  The code reads better if one form of concatenation is used instead of two (here you are using both `+` and `${}`).